docs/docs/gestures/gesture-detector.md

```diff
@@ -47,3 +47,25 @@ This parameter allows to specify which `userSelect` property should be applied t
 - Gesture Detector will use first native view in its subtree to recognize gestures, however if this view is used only to group its children it may get automatically [collapsed](https://reactnative.dev/docs/view#collapsable-android). Consider this example:
   <FunctionalComponents />
   If we were to remove the collapsable prop from the View, the gesture would stop working because it would be attached to a view that is not present in the view hierarchy. Gesture Detector adds this prop automatically to its direct child but it's impossible to do automatically for more complex view trees.
+
+- Using the same instance of gesture handler across multiple Gesture Detectors is not possible. Have a look at the code below:
```

Comment:
```suggestion
- Using the same instance of a gesture across multiple Gesture Detectors is not possible. Have a look at the code below:
```

Reply:
Done in [0b178e3](https://github.com/software-mansion/react-native-gesture-handler/pull/2694/commits/0b178e350481ae074caccd46657da242d2f48f65)